edgegrid/auth.go

```diff
@@ -13,6 +13,9 @@ import (
 	"github.com/satori/go.uuid"
 )
 
+// AuthParams is used to house various request details such that
+// the AuthParams{} can be passed to Auth to sign using the
+// Akamai {OPEN} EdgeGrid Authentication scheme
```

Comment:
Documentation comments such as this should end with a period (.). Many more below.